UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -694,6 +694,87 @@ public void UpdateClientBodyPartStats(BodyPartType bodyPartType, float bruteDama
 		}
 	}
 
+	/// ---------------------------
+	/// Electrocution Methods
+	/// ---------------------------
+	/// Note: Electrocution for players is extended in PlayerHealth deriviative.
+	/// This is a generic electrocution implementation that just deals damage.
+
+	/// <summary>
+	/// Electrocutes a mob, applying damage to the victim depending on the electrocution power.
+	/// </summary>
+	/// <param name="electrocution">The object containing all information for this electrocution</param>
+	/// <returns>Returns an ElectrocutionSeverity for when the following logic depends on the elctrocution severity.</returns>
+	public virtual ElectrocutionSeverity Electrocute(Electrocution electrocution)
+	{
+		float resistance = ApproximateElectricalResistance(electrocution.Voltage);
+		electrocution.shockPower = Electrocution.CalculateShockPower(electrocution.Voltage, resistance);
```

Comment:
I think this just shows how severity can just be part of the internal implementation in LHB / PlayerHealth. Just check the range and call a different virtual method depending on the severity, IDK if you even need an enum.